templates/profile.html

```diff
@@ -72,10 +71,49 @@ <h1>{{ title }}</h1>
       </div>
       </div>
     {% else %}
-
       {% if profile %}
         <table class=table>
           <tbody>
+            <tr>
+              <th>Edycje WWW:</th>
+              <td>
+                {% for edition in profile.workshop_profile.all %}
+                  {% if edition.status == 'Z' %}
+                    <span class="qualified">✔{{ edition.year }}</span>
+                  {% else %}
+                    <span class="not-qualified">✘{{ edition.year }}</span>
+                  {% endif %}
+                {% endfor %}
+                {% if has_workshop_profile %}
+                  <form method="post" class="inline">
+                    {% csrf_token %}
+                    <span class="btn-group-xs" role="group">
+                      <button type="submit" class="btn btn-default"
+                              name="qualify" value="delete"
+                              onclick="return confirm('Czy na pewno chcesz usunąć kwalifikację?');">
```

Comment:
https://stackoverflow.com/questions/8982295/confirm-deletion-in-modal-dialog-using-twitter-bootstrap